Firestore/core/src/firebase/firestore/util/string_apple.mm

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Firestore/core/src/firebase/firestore/util/string_apple.h"
+
+#import <Foundation/Foundation.h>
+
+#include <string>
+
+namespace firebase {
+namespace firestore {
+namespace util {
+
+NSString* WrapNSStringNoCopy(const char* c_str) {
```

Comment:
Given that this is just wrapping this call to the NSString initializer, should we make this implementation `inline` in the header?

Reply:
done